wolfcrypt/src/pkcs12.c

```diff
@@ -1012,18 +1012,25 @@ void wc_FreeCertList(WC_DerCertList* list, void* heap)
     (void)heap;
 }
 
-static void freeDecCertList(WC_DerCertList** list, byte** pkey, word32* pkeySz,
-    byte** cert, word32* certSz, void* heap)
+static WARN_UNUSED_RESULT int freeDecCertList(WC_DerCertList** list,
```

Comment:
Is `WARN_UNUSED_RESULT` still required?

Reply:
`WARN_UNUSED_RESULT` is never required, but with the change I made for `_SMALL_STACK`, it is always an error to ignore the return value from `freeDecCertList()`, which previously was a `void` function.  this is surprising, so I used `WARN_UNUSED_RESULT` as a programmer aid.